plugins/modules/postgresql_info.py

```diff
@@ -925,12 +943,25 @@ def get_pg_version(self):
         """Get major and minor PostgreSQL server version."""
         query = "SELECT version()"
         raw = self.__exec_sql(query)[0][0]
-        raw = raw.split()[1].split('.')
+        full = raw.split()[1]
+        tmp = full.split('.')
+
+        major = int(tmp[0])
+        minor = int(tmp[1].rstrip(','))
+        patch = None
+        if len(tmp) == 3:
```

Comment:
How about
```suggestion
        if len(tmp) >= 3:
```
in case there's ever another component?

Reply:
sounds sensible, in case of something new is introduced later (I hope it won't be but anyway). committed, thanks:)